XSUP 39659 - DUO event collector (#35588)
demisto/content: Packs/DuoAdminApi/Integrations/DuoEventCollector/README.md

#### Context Output

There is no context output for this command.

## Additional information

> * The Duo eventing system is not real-time. It takes a few minutes for the events to be indexed and available for an API call due to consolidation. This is why we added the parameter "End of the fetch window" to adjust XSIAM to Duo's delay.

Reviewer: ```suggestion
* The Duo eventing system is not real-time. It takes a few minutes for the events to be indexed and available for an API call due to consolidation. As a result parameter "End of the fetch window" to adjust XSIAM to Duo's delay was added.
```
